app/controllers/challenges_controller.rb

```diff
@@ -19,6 +20,26 @@ def create
     end
   end
 
+  def for_evaluation
+    @evaluable_challenges = Challenge.evaluable(current_user)
+
+    if @evaluable_challenges.any?
+      render :for_evaluation, status: :ok
+    else
+      render json: { resource: 'challenge', errors: [challenge: 'not found'] }, status: :not_found
```

Comment:
Yea when I wrote it I was thinking "well if nothing is returned it means they are auth'd to see them" instead of "maybe there just aren't any to evaluate"